### DISCUSSION TITLE
请问用这个rec_svtrnet_ch.yml训练出来的模型可以转成onnx模型吗？有现成的工具吗

### DISCUSSION THREAD

**Discussion by ctgushiwei** [PaddlePaddle/PaddleOCR#12964] [Q&A]

**Comment by a31413510** [comment]
您好，可以使用Paddle2ONNX，可以参考这个文档
https://github.com/PaddlePaddle/PaddleOCR/blob/52cf1e1bdd2a4422bd6232e924bf26b85a47afda/deploy/paddle2onnx/readme_ch.md
